docs/pages/guides/using-push-notifications-services.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Using Push Notifications
```

Comment:
```suggestion
title: Using push notifications
```

We use sentence cases instead of title case unless there is a brand/feature/product name involved.